visualizations/frontend/other/TTable.vue

```diff
@@ -380,6 +376,38 @@ export default {
     uniqueOriginalRows() {
       return _.uniq(this.internalRows.map((r) => r.originalRow));
     },
+    layerRows() {
+      if (typeof this.rows === "function")
+        return this.rows(this.$attrs?.["t-layer"]);
+      if (this.rows) return this.rows;
+      return [];
+    },
+  },
+  watch: {
+    loading() {
+      if (!this.loading) {
+        this.updateEndIndex(this.rowsPerPage);
+        this.updateStartIndex(0);
+        this.pagerResetFunction();
+        this.fetchTotalRows();
+      }
+      this.showSpinner = this.loading;
+    },
+    internalSelectedItem() {
+      this.$emit("update:selectedItem", this.internalSelectedItem);
+      this.$emit("selectedItemChanged", this.internalSelectedItem);
```

Comment:
Why do we need to emit two events that looks the same?

Reply:
This is from pre-topcoat code. The original table is meant to be very flexible, so most users will want to use the .sync modifier that will work with the first line, but there are times when that won't work/be sufficient so users can also react to the event instead.